shared/js/background/atb.js

```diff
@@ -175,6 +243,11 @@ const ATB = (() => {
                         return !!atb
                     })
 
+                    // in case there is no assigned atb, enroll into Chrome Counteract experiment
```

Comment:
Should we be filtering out non chrome users?

Reply:
yes, updated!